### PR TITLE
Feature/edit and delete tasks

### LINKED ISSUE #18: Edit and delete existing Kapacitor tasks

### PR DESCRIPTION
**Pull request by hefowe**
Closes #18

### CODE REVIEW

File: src/apis/KapacitorApi.ts [hunk 1]
```diff
@@ -16,10 +18,10 @@ export default class KapacitorApi {
         return test ? test.level : 'OK';
     }
 
-    public static async taskVariables(taskName: string): Promise<any> {
-        const response = await fetch(`http://82.140.0.78:9092/kapacitor/v1/tasks/${taskName}`);
-        const topic = await response.json();
-        return topic.vars;
+    public static async tasks(): Promise<KapacitorTask[]> {
```
Comment (friedow) [inline src/apis/KapacitorApi.ts:21]: The return type should be specified through an interface.

File: src/components/TaskVariablesModal.vue [hunk 2]
```diff
@@ -52,6 +61,8 @@ export default class TaskVariablesModal extends Vue {
   // endregion
 
   // region private members
+  private tasks: any = {};
```
Comment (friedow) [inline src/components/TaskVariablesModal.vue]: The type should be specified through an interface.

File: src/components/TaskVariablesModal.vue [hunk 3]
```diff
@@ -52,6 +61,8 @@ export default class TaskVariablesModal extends Vue {
...
+  private selectedTaskName = this.taskName;
   private taskVariables: any = {};
```
Comment (friedow) [inline src/components/TaskVariablesModal.vue]: The type should be specified through an interface.

**Review by friedow — CHANGES_REQUESTED** [review]

File: src/apis/KapacitorApi.ts (hunk 1 above)
Comment (hefowe) [reply src/apis/KapacitorApi.ts:21]: What do you mean exactly? `KapacitorTask[]` is the interface of the return type.

**Comment by hefowe** [comment]
@friedow Please take another look

**Review by friedow — APPROVED** [review]

**Pull request merged**